platform/parsers/elastic_query_dsl/pancake_json_rendering.go

```diff
@@ -246,12 +263,33 @@ func (p *pancakeJSONRenderer) layerToJSON(remainingLayers []*pancakeModelLayer,
 	}
 
 	layer := remainingLayers[0]
-
+	fmt.Println("kk model:", rows)
 	for _, metric := range layer.currentMetricAggregations {
 		var metricRows []model.QueryResultRow
 		switch metric.queryType.(type) {
 		case *metrics_aggregations.TopMetrics, *metrics_aggregations.TopHits:
 			metricRows = p.selectTopHitsRows(metric, rows)
+		case *metrics_aggregations.Rate:
+			// Special, as we need to select also parent date_histogram's values.
+
+			// 2 lines below: e.g. metric__2__year -> aggr__2
+			parentHistogramColName := fmt.Sprintf("aggr%s", strings.TrimPrefix(metric.internalName, "metric"))
+			parentHistogramColName = strings.TrimSuffix(parentHistogramColName, metric.name)
+
+			var (
+				parentHistogramKey = fmt.Sprintf("%skey_0", parentHistogramColName)
```

Comment:
Not sure if I like this logic, instead we should pass previous bucket aggregation, if exists and grab values from it.